index.ts

```diff
@@ -226,11 +255,13 @@ Clear all cached data of a memoized function.
 export function memoizeClear(function_: AnyFunction): void {
 	const cache = cacheStore.get(function_);
 	if (!cache) {
-		throw new TypeError('Can\'t clear a function that was not memoized!');
+		// eslint-disable-next-line @typescript-eslint/quotes
```

Comment:
The lint rule should not be ignored.

Reply:
Got it, my formatter seems pretty opinionated - is there a linter / formatter configuration that I'm missing / not activating?